app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float,
+                 clean_power: int, average_rating: float,
+                 count_of_ratings: int) -> None:
+        if distance_from_city_center <= 0:
+            raise ValueError(
+                "Distance from city center must be greater than 0"
+            )
```

Comment:
The check for `distance_from_city_center` being less than or equal to zero is correctly implemented to prevent division by zero in the `calculate_washing_price` method. Ensure this check is maintained to avoid runtime errors.